### ISSUE TITLE
ffmpeg exited with code 1

### ISSUE THREAD

**Issue by ndugger** [fluent-ffmpeg/node-fluent-ffmpeg#478]
- OSX 10.10.5
- fluent-ffmpeg 2.0.1
- ffmpeg 2.8.2

For some reason, I can use fluent-ffmpeg on only one specific video file; It's only about 31 seconds long, so I'm wondering if this is maybe an issue of file size?

I'm POSTing a file/buffer to my server, and using `new stream.Readable` to essentially convert my Buffer into a Stream.

I then pass this stream into ffmpeg, and it works beautifully if I use the smaller video file, but when I try to use one that's only about a minute long (entirely different video file, made by someone else)--and any other videos--it throws an error, and fails.

Here is my code:
```javascript
const video = new stream.Readable({
	read () {
		this.push(myVideoBuffer);
		this.push(null);
	}
});

ffmpeg(video)
	.on('progress', e => console.log(e))
	.on('end', () => {
		console.log('ended')
	})
	.on('error', e => console.error(e))
	.save(`${process.cwd()}/media/videos/${Date.now()}.mp4`);
```

All of the videos I'm testing with are already MP4 files, so it's not doing any format conversion, it should just be saving my POSTed Buffer in my videos directory, yes?

the "**stdout**" is empty, but here's my "**stderr**":

```
ffmpeg version 2.8.2 Copyright (c) 2000-2015 the FFmpeg developers
  built with llvm-gcc 4.2.1 (LLVM build 2336.11.00)
  configuration: --prefix=/Volumes/Ramdisk/sw --enable-gpl --enable-pthreads --enable-version3 --enable-libspeex --enable-libvpx --disable-decoder=libvpx --enable-libmp3lame --enable-libtheora --enable-libvorbis --enable-libx264 --enable-avfilter --enable-libopencore_amrwb --enable-libopencore_amrnb --enable-filters --enable-libgsm --enable-libvidstab --enable-libx265 --disable-doc --arch=x86_64 --enable-runtime-cpudetect
  libavutil      54. 31.100 / 54. 31.100
  libavcodec     56. 60.100 / 56. 60.100
  libavformat    56. 40.101 / 56. 40.101
  libavdevice    56.  4.100 / 56.  4.100
  libavfilter     5. 40.101 /  5. 40.101
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  2.101 /  1.  2.101
  libpostproc    53.  3.100 / 53.  3.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'pipe:0':
  Metadata:
    major_brand     : mp42
    minor_version   : 512
    compatible_brands: mp42
    creation_time   : 2015-12-01 19:57:03
    media_type      : 9
    season_number   : 1
    episode_sort    : 1
    hd_video        : 0
    iTunMOVI        : <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN"  "http://www.apple.com/DTDs/PropertyList-1.0.dtd"><plist version="1.0"><dict><key>studio</key><string>studio</string><key>cast</key><array><dict><key>name<
  Duration: 00:01:01.44, start: 0.000000, bitrate: N/A
    Stream #0:0(eng): Video: h264 (Constrained Baseline) (avc1 / 0x31637661), yuv420p, 640x360 [SAR 1:1 DAR 16:9], 927 kb/s, 24 fps, 24 tbr, 90k tbn, 48 tbc (default)
    Metadata:
      creation_time   : 2015-12-01 19:57:03
      handler_name    : VideoHandler
      encoder         : h264
    Stream #0:1(eng): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 117 kb/s (default)
    Metadata:
      creation_time   : 2015-12-01 19:57:03
      handler_name    : SoundHandler
[libx264 @ 0x7f9581803800] using SAR=1/1
[libx264 @ 0x7f9581803800] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 0x7f9581803800] profile High, level 3.0
[libx264 @ 0x7f9581803800] 264 - core 146 - H.264/MPEG-4 AVC codec - Copyleft 2003-2015 - http://www.videolan.org/x264.html - options: cabac=1 ref=3 deblock=1:0:0 analyse=0x3:0x113 me=hex subme=7 psy=1 psy_rd=1.00:0.00 mixed_ref=1 me_range=16 chroma_me=1 trellis=1 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=-2 threads=12 lookahead_threads=2 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=2 keyint=250 keyint_min=24 scenecut=40 intra_refresh=0 rc_lookahead=40 rc=crf mbtree=1 crf=23.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=1:1.00
[aac @ 0x7f9581804a00] The encoder 'aac' is experimental but experimental codecs are not enabled, add '-strict -2' if you want to use it.
```

**Comment by ndugger** [comment]
I'm gonna leave this here until It's seen, but I was able to fix it by looking at that last line in the stderr (I really need to pay closer attention).

I added `outputOptions('-strict -2')` to my ffmpeg chain, and it worked. I got a pre-built binary of ffmpeg, and I read in their docs that due to licensing, the pre-built versions do not come with the other encoders, and this one, I guess, was disabled by default.

https://trac.ffmpeg.org/wiki/Encode/AAC
